packages/shared/providers.yaml

```diff
@@ -5675,6 +5675,49 @@ rapidapi:
             suffix: .p.rapidapi.com
             prefix: https://
 
+readwise:
+    display_name: Readwise
+    categories:
+        - productivity
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://readwise.io/api/v2
+        headers:
+            authorization: Token ${apiKey}
+        verification:
+            method: GET
+            endpoint: /auth
+        retry:
+            after: 'retry-after'
+    docs: https://docs.nango.dev/integrations/all/readwise
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your Readwise account
+            
+readwise-reader:
+    display_name: Readwise Reader
+    categories:
+        - productivity
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://readwise.io/api/v3
+        headers:
+            authorization: Token ${apiKey}
+        verification:
+            method: GET
+            endpoint: https://readwise.io/api/v2/auth
```

Comment:
the v3 api still used the v2/auth for verification, is it ok to have the full url like this? or will it lead to a bug i.e. `https://readwise.io/api/v3/https://readwise.io/api/v2/auth` ?

Reply:
good question, we need to check

Reply:
I tried running Nango locally. Readwise works but Readwise Reader does not work

![CleanShot 2025-01-11 at 12 52 06@2x](https://github.com/user-attachments/assets/e5836695-5ebd-4ecb-98d3-df05351cf42b)

Perhaps someone from the team would be willing to make a change that allows the endpoint to be a full URL or point me to the code, I am happy to take a stab at it

Reply:
From reading the code I think on the verification block you can set `baseUrlOverride` and it will use that on verify.

See https://github.com/NangoHQ/nango/blob/b61728bf82e27631eb9ec73fd9b6e771ec9f38dd/packages/server/lib/hooks/hooks.ts#L229